index.js

```diff
@@ -83,7 +83,13 @@ Configstore.prototype.get = function (key) {
 
 Configstore.prototype.set = function (key, val) {
 	var config = this.all;
-	config[key] = val;
+	if (key instanceof Object) {
```

Comment:
`instanceof Object` can fail in many scenarios. Better to just check `arguments.length`.